app/javascript/modCenter/singleArticle/index.jsx

```diff
@@ -1,88 +1,76 @@
-    );
-  });
-
-  const months = [
-    'Jan',
-    'Feb',
-    'Mar',
-    'Apr',
-    'May',
-    'Jun',
-    'Jul',
-    'Aug',
-    'Sep',
-    'Oct',
-    'Nov',
-    'Dec',
-  ];
+export default class SingleArticle extends Component {
+  constructor(props) {
+    super(props);
 
-  const get12HourTime = (date) => {
-    const minutes = date.getMinutes();
-    let hours = date.getHours();
-    const AmOrPm = hours >= 12 ? 'PM' : 'AM';
-    hours = hours % 12 || 12;
+    this.state = {
+      articleOpened: false,
+    };
+  }
 
-    return `${hours}:${minutes} ${AmOrPm}`;
-  };
-
-  const formatDate = (timestamp) => {
-    const dateToday = new Date();
-    const origDatePublished = new Date(timestamp);
+  toggleArticle = (e) => {
+    e.preventDefault();
 
-    if (dateToday.toDateString() === origDatePublished.toDateString()) {
-      return get12HourTime(origDatePublished);
+    const { id, path } = this.props;
+    const { articleOpened } = this.state;
+    if (articleOpened) {
+      this.setState({ articleOpened: false });
+      document.getElementById(`article-iframe-${id}`).innerHTML = '';
+    } else {
+      this.setState({ articleOpened: true });
+      document.getElementById(
+        `article-iframe-${id}`,
+      ).innerHTML = `<iframe class="article-iframe" src="${path}"></iframe><iframe class="actions-panel-iframe" src="${path}/actions_panel"></iframe>`;
     }
-    return `${
-      months[origDatePublished.getMonth()]
-    } ${origDatePublished.getDate()}`;
   };
 
-  const newAuthorNotification = () => {
-    if (user.articles_count <= 3) {
-      return '👋 ';
-    }
-    return '';
-  };
+  render() {
+    const { id, title, publishedAt, cachedTagList, user } = this.props;
+    const tags = cachedTagList.split(', ').map((tag) => {
+      return (
+        <span className="mod-article-tag">
+          <span className="article-hash-tag">#</span>
+          {tag}
+        </span>
+      );
+    });
 
-  return (
-    <div className="moderation-single-article">
-      <span className="article-title">
-        <header>
-          <h3>{title}</h3>
-        </header>
-        {tags}
-      </span>
-      <span className="article-author">
-        {newAuthorNotification()}
-        {user.name}
-      </span>
-      <span className="article-published-at">{formatDate(publishedAt)}</span>
-    </div>
-  );
-};
+    const newAuthorNotification = user.articles_count <= 3 ? '👋 ' : '';
+
+    return (
+      <button
+        type="button"
+        className="moderation-single-article"
+        onClick={this.toggleArticle}
+      >
+        <span className="article-title">
+          <header>
+            <h3>{title}</h3>
+          </header>
+          {tags}
+        </span>
+        <span className="article-author">
+          {newAuthorNotification}
+          {user.name}
+        </span>
+        <span className="article-published-at">{formatDate(publishedAt)}</span>
+        <div
+          className="article-iframes-container"
```

Comment:
A few comments:

1. I think the `iframe` border looks clunky 😟 We could set `frameBorder="0"` on the `iframe` element, which would remove the border completely, but then the iframes look like they're floating (🚫). We could then add `border: 1px solid var(--bg-color)` which gives the iframes a subtle border, but then we'd need to hide this border when the article is collapsed. So perhaps we leave it as-is for now. Just looks clunky...

2. This is minor, but the snackbar confirmations appear way down the article, and will probably not be seen by the mod (unless the article is super-short). Do we care? I really don't